src/CSFeeDistributor.sol

```diff
@@ -90,16 +96,18 @@ contract CSFeeDistributor is
         bytes32 _treeRoot,
         string calldata _treeCid,
         string calldata _logCid,
-        uint256 distributed
+        uint256 _distributedShares,
```

Comment:
would suggest to not to rename it to keep consistency here: `DistributionData` has `distributed`. moreover, we already have `distributedShares` and it is mapping of shares per node operator, so the new name can mislead a bit

Reply:
Done